src/Models/LoginSession.php

```diff
@@ -96,6 +97,14 @@ class LoginSession extends DataObject
      */
     private static $default_session_lifetime = 3600;
 
+    /**
+     * The length of time between two updates to the LastAccessed field
+     *
+     * @config
+     * @var int
+     */
+    private static $last_accessed_threshold = 300;
```

Comment:
```suggestion
     */
    private static int $last_accessed_threshold = 300;
```
We prefer just explicitly typehinting for newly added configuration properties (even though the config system doesn't _currently_ respect it, it's still our preferred way to document the type and the typehint could be respected by the config system in the future)